content/en/docs/concepts/configuration/secret.md

```diff
@@ -135,8 +107,8 @@ Here are some of your options:
   [ServiceAccount](/docs/reference/access-authn-authz/authentication/#service-account-tokens)
   and its tokens to identify your client.
 - There are third-party tools that you can run, either within or outside your cluster,
-  that provide secrets management. For example, a service that Pods access over HTTPS,
-  that reveals a secret if the client correctly authenticates (for example, with a ServiceAccount
+  that manage secret data. For example, a service that Pods access over HTTPS,
```

Comment:
[nit] `manage Secret data` capitalization missing

Reply:
I read "secret" here as "sensitive data" and not "data in a Secret" but maybe that's a wrong assumption

Reply:
I changed the two instances of "secret data" to "sensitive data" to avoid ambiguity

Reply:
Thank you!